### PR TITLE
ABC-123 : New handing for broeser

### PR DESCRIPTION
**Pull request by Akash07y**

### CODE REVIEW

**Review by shivaflash — APPROVED** [review]
all ok

File: OrangeCRM/src/test/java/base/Browser.java [hunk 1]
```diff
@@ -27,4 +27,10 @@ public static WebDriver launchEdge() {
 		WebDriver driver  = new EdgeDriver();	
 		return driver ;
 	}
+
+	public static WebDriver launchSafari() {
```
Comment (Akash07y) [inline OrangeCRM/src/test/java/base/Browser.java:31]: could you please change the method name

File: OrangeCRM/src/test/java/tests/TestClass2.java [hunk 2]
```diff
@@ -27,10 +27,6 @@ public void beforeSuite() {
 			System.out.println("Before Suite TestClass222");
 		}
 		
-		@BeforeTest
-		public void beforeTest() {
```
Comment (Akash07y) [inline OrangeCRM/src/test/java/tests/TestClass2.java:31]: you can not delete this

**Review by Akash07y — COMMENTED** [review]
All ok

File: OrangeCRM/src/test/java/base/Browser.java (hunk 1 above)
Comment (Sangitachaudhar) [reply OrangeCRM/src/test/java/base/Browser.java:31]: all fine

File: OrangeCRM/src/test/java/base/Browser.java [hunk 3]
```diff
@@ -27,4 +27,10 @@ public static WebDriver launchEdge() {
 		WebDriver driver  = new EdgeDriver();	
 		return driver ;
 	}
+
+	public static WebDriver launchSafari() {
+		System.setProperty("webdriver.safari.driver", "src\\test\\resources\\BrowserFiles\\safari.exe");
+		WebDriver driver  = new SafariDriver();
```
Comment (KalyaniRathod) [inline OrangeCRM/src/test/java/base/Browser.java:33]: safari browser

File: OrangeCRM/src/test/java/tests/TestClass2.java [hunk 4]
```diff
@@ -46,6 +42,12 @@ public void test1() {
 			System.out.println("Test A TestClass222");
 				
 		}
+
+		@Test (groups = {"High Priority", "Sanity"})
+		public void test12344() {
+			System.out.println("Test A TestClass343545");
```
Comment (ghost) [inline OrangeCRM/src/test/java/tests/TestClass2.java:48]: hi

File: OrangeCRM/src/test/java/base/Browser.java (hunk 1 above)
Comment (Sangitachaudhar) [inline OrangeCRM/src/test/java/base/Browser.java:31]: code is ok

File: OrangeCRM/src/test/java/base/Browser.java (hunk 3 above)
Comment (Akash07y) [reply OrangeCRM/src/test/java/base/Browser.java:33]: Resoved

**Review by ghost — APPROVED** [review]

File: OrangeCRM/src/test/java/base/Browser.java [hunk 5]
```diff
@@ -27,4 +27,10 @@ public static WebDriver launchEdge() {
 		WebDriver driver  = new EdgeDriver();	
 		return driver ;
 	}
+
```
Comment (Pankajsahu197) [inline OrangeCRM/src/test/java/base/Browser.java:30]: Remove edge browser.

**Review by Akash07y — COMMENTED** [review]
all ok

**Review by Sangitachaudhar — APPROVED** [review]
approve

**Review by KalyaniRathod — APPROVED** [review]
all ok

**Review by MauliBalwadkar — APPROVED** [review]
ok

**Review by ghost — APPROVED** [review]

**Review by Pankajsahu197 — APPROVED** [review]
Approved

File: OrangeCRM/src/test/java/base/Browser.java (hunk 1 above)
Comment (Vishakha8329) [inline OrangeCRM/src/test/java/base/Browser.java:31]: could you Please change the browser name?

File: OrangeCRM/src/test/java/tests/TestClass2.java [hunk 6]
```diff
@@ -46,6 +42,12 @@ public void test1() {
 			System.out.println("Test A TestClass222");
 				
 		}
+
+		@Test (groups = {"High Priority", "Sanity"})
+		public void test12344() {
```
Comment (Vishakha8329) [inline OrangeCRM/src/test/java/tests/TestClass2.java:47]: ok

**Review by Vishakha8329 — COMMENTED** [review]
ok

**Pull request merged**